corehq/motech/fhir/urls.py

```diff
@@ -18,4 +19,7 @@
         name="smart_configuration_view"
     ),
     url(r'^fhir/(?P<fhir_version_name>\w+)/metadata', smart_metadata_view, name="smart_metadata_view"),
+
+
+    url(r'^oauth/login', SmartAuthView.as_view(), name=SmartAuthView.urlname),
```

Comment:
why do all the other urls start with `fhir` but this one? is this url namespace imported at the root in base URLs? just thinking we don't want this to be the *only* way to oauth login if it's set up that way.

Reply:
This URL is still domain based, so the original OAuth routes are still available (those you use for the API explorer, for instance). 
These become something like `https://staging.commcarehq.org/a/smart-on-fhir-poc/oauth/login`
I maybe didn't give this enough thought, but in my mind having these a "level up" from the routes that were getting requested made sense, if we ever use domain based OAuth for something else... but now I'm questioning that decision.

Reply:
I'm pro-namespace even below the domain. Seems low downside, high upside. (see also: couch case properties)